moveit_core/collision_detection_bullet/src/collision_world_bt.cpp

```diff
@@ -84,22 +105,71 @@ void CollisionWorldBt::checkRobotCollision(const CollisionRequest& req, Collisio
                                            const CollisionRobot& robot, const robot_state::RobotState& state1,
                                            const robot_state::RobotState& state2) const
 {
-  ROS_ERROR_NAMED("collision_detection.bullet", "Bullet continuous collision checking not yet implemented");
+  checkRobotCollisionHelperCCD(req, res, robot, state1, state2, nullptr);
 }
 
 void CollisionWorldBt::checkRobotCollision(const CollisionRequest& req, CollisionResult& res,
                                            const CollisionRobot& robot, const robot_state::RobotState& state1,
                                            const robot_state::RobotState& state2,
                                            const AllowedCollisionMatrix& acm) const
 {
-  ROS_ERROR_NAMED("collision_detection.bullet", "Bullet continuous collision checking not yet implemented");
+  checkRobotCollisionHelperCCD(req, res, robot, state1, state2, &acm);
+}
+
+void CollisionWorldBt::checkRobotCollisionHelperCCD(const CollisionRequest& req, CollisionResult& res,
+                                                    const CollisionRobot& robot, const robot_state::RobotState& state1,
+                                                    const robot_state::RobotState& state2,
+                                                    const AllowedCollisionMatrix* acm) const
+{
+  const CollisionRobotBt& robot_bt = dynamic_cast<const CollisionRobotBt&>(robot);
+
+  tesseract::BulletCastBVHManagerPtr temp_clone_manager = bt_manager_CCD_->clone();
```

Comment:
```suggestion
  tesseract::BulletCastBVHManagerPtr bvh_clone_manager = bt_manager_CCD_->clone();
```
what does temp mean here?

Reply:
The clone() mostly takes place because all the collision checking function are `const`. Therefore, the cloned one is only used in single collision check, that is why I called it temp. But I agree, all local variables are "temp", so I'll rename it.